docs/api.md

```diff
@@ -200,6 +200,13 @@ Indicates that the browser is connected.
     - `deviceScaleFactor` <[number]> Specify device scale factor (can be thought of as dpr). Defaults to `1`.
     - `isMobile` <[boolean]> Whether the `meta viewport` tag is taken into account and touch events are enabled. Defaults to `false`. Not supported in Firefox.
   - `userAgent` <?[string]> Specific user agent to use in this context.
+  - `device` <[string]|[Object]> Pass a device name or object from [playwright.devices](#playwrightdevices) to set the viewport and user agent.
```

Comment:
may we only have a "string" here as value?

Reply:
I like that we abstract the `DeviceDescriptor` object, because I can envision it having more than just `userAgent` and `viewport` in the future. I'm fine with replacing the device with just a string here, but I think we should remove `playwright.devices` then, or at least remove the examples encouraging people to manually extract viewport and userAgent from it.